### DISCUSSION TITLE
Difference between 1.x vs 2.x java agent

### DISCUSSION THREAD

**Discussion by jlmanohar** [open-telemetry/opentelemetry-java-instrumentation#11964] [Q&A] [answered]
I am trying to add Open telemetry agent into our stack, along with zipkin and trying to understand what exactly is the difference between agent V1 and V2, and why should I choose one over another

We use Java 1.8 and apps are deployed on jboss / wildfly

Any help is appreciated

**Comment by jaydeluca** [comment, marked as answer]
> what exactly is the difference between agent V1 and V2

The 2.0 release included significant breaking changes, and thus justified a major version bump. You can find more details in [the release notes](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v2.0.0).

>  why should I choose one over another

It would be recommended to use the latest 2.x release, as it will have the latest features and improvements. 1.32.x has kept up with security patches but does not include other bug fixes and enhancements. Perhaps a reason to use 1.x would be if you had some dependency on the previous semantic conventions (in dashboards or monitors for example), but if you are starting fresh, use the latest version.

**Reply by SubieGold** [reply]
Can this be added to the README? I'm trying to find this answer for a while.

**Reply by jaydeluca** [reply]
@LoganDev99 I've opened a pr here: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/12239
